pkg/constant/constant_shared.go

```diff
@@ -72,7 +72,7 @@ const (
 	/* Image Constants */
 
 	KonnectivityImage                  = "quay.io/k0sproject/apiserver-network-proxy-agent"
-	KonnectivityImageVersion           = "0.0.33-k0s.1"
+	KonnectivityImageVersion           = "v0.1.4"
```

Comment:
this version was built from our private image builder using the upstream source repo, unlike the previous one, which was built from the forked repo